validators/crypto_addresses/eth_address.py

```diff
@@ -0,0 +1,57 @@
+"""ETH Address."""
+# -*- coding: utf-8 -*-
+
+# standard
+import re
+
```

Comment:
```diff
+                                                             
+ # external
```

ref: https://github.com/python-validators/validators/pull/276/files#diff-bf12d04b521219b077ca524ec38eff66759c3c457e297089f6d40329f80102c6R4

Reply:
@joe733  Done. Thanks for the tip

https://github.com/python-validators/validators/pull/276/commits/b40cb12b3f7cdb915f4efc3956d3a2bb13150fe8

Reply:
@msamsami, I meant the change to be as follows:

```py
# standard
import re

# external
from eth_hash.auto import keccak

# local
from validators.utils import validator
```

As `re` is from the _standard_ library, `eth_hash.auto.keccak` is an _external_ dependency and `validators.utils.validator` implies a _local_ coupling.